htlcswitch/link.go

```diff
@@ -3207,7 +3210,7 @@ func (l *channelLink) getAliases() []lnwire.ShortChannelID {
 // attachFailAliasUpdate sets the link's FailAliasUpdate function.
 //
 // Part of the scidAliasHandler interface.
-func (l *channelLink) attachFailAliasUpdate(closure func(
+func (l *channelLink) attachFailAliasUpdate(closure func(ctx context.Context,
```

Comment:
Then I think we can create a `ctx` via `l.cg.Create(context.Background())` and pass it to the closure below so it still satisfies the signature `failAliasUpdate`, think will bring way less change, and create fewer conflicts in other people's PRs.